src/stream/src/executor/top_n_executor.rs

```diff
@@ -48,6 +49,10 @@ pub trait TopNExecutorBase: Send + 'static {
     /// See [`Executor::identity`].
     fn identity(&self) -> &str;
 
+    /// Update the vnode bitmap for the state tables, only used by Group Top-N since it's
+    /// distributed.
+    fn update_state_table_vnode_bitmap(&mut self, _vnode_bitmap: Arc<Bitmap>) {}
```

Comment:
Since only group TopN uses this, why not just impl for group_top_n?

Reply:
The `execute` is actually implemented by `TopNExecutorWrapper` and there's no way to know which inner `base` it is. So we need a generic interface for this. 🤔